sgv2-restapi/src/main/java/io/stargate/sgv2/restsvc/resources/schemas/Sgv2ColumnsResourceImpl.java

```diff
@@ -0,0 +1,282 @@
+package io.stargate.sgv2.restsvc.resources.schemas;
+
+import io.stargate.proto.QueryOuterClass;
+import io.stargate.proto.Schema;
+import io.stargate.proto.StargateGrpc;
+import io.stargate.sgv2.common.cql.builder.Column;
+import io.stargate.sgv2.common.cql.builder.ImmutableColumn;
+import io.stargate.sgv2.common.cql.builder.QueryBuilder;
+import io.stargate.sgv2.restsvc.grpc.BridgeProtoTypeTranslator;
+import io.stargate.sgv2.restsvc.models.Sgv2ColumnDefinition;
+import io.stargate.sgv2.restsvc.models.Sgv2RESTResponse;
+import io.stargate.sgv2.restsvc.resources.CreateGrpcStub;
+import io.stargate.sgv2.restsvc.resources.ResourceBase;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import javax.inject.Singleton;
+import javax.servlet.http.HttpServletRequest;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Path("/v2/schemas/keyspaces/{keyspaceName}/tables/{tableName}/columns")
+@Produces(MediaType.APPLICATION_JSON)
+@Singleton
+@CreateGrpcStub
+public class Sgv2ColumnsResourceImpl extends ResourceBase implements Sgv2ColumnsResourceApi {
+  // Singleton resource so no need to be static
+  protected final Logger logger = LoggerFactory.getLogger(getClass());
+
+  @Override
+  public Response getAllColumns(
```

Comment:
OK I might be missing something around the authorization I guess.. where is it done?

Reply:
by gRPC endpoint that is used for all Persistence access.